src/components/RepaymentScheduleGraph.tsx

```diff
@@ -0,0 +1,37 @@
+import { Column } from "@ant-design/plots";
+import { FunctionComponent } from "react";
+
+export enum RepaymentScheduleAmountType {
+	INTEREST = "interest",
+	PRINCIPAL = "principal",
+}
+
+export interface RepaymentScheduleDataPoint {
+	month: string;
+	type: RepaymentScheduleAmountType;
+	amount: number;
+}
+
+interface RepaymentScheduleGraphProps {
+	data: RepaymentScheduleDataPoint[];
+}
+
+export const RepaymentScheduleGraph: FunctionComponent<RepaymentScheduleGraphProps> = ({
+	data,
+}) => {
+	const config = {
+		data,
+		height: 200,
+		width: 320,
+		isStack: true,
+		xField: "month",
```

Comment:
use intl

Reply:
The "month" refers to an object property, `intl` can't be used in this context.